### PR TITLE
[MRG] fix time_unit in evoked plots

### LINKED ISSUE #5235: evoked.plot displays ms instad of s

### PR DESCRIPTION
**Pull request by massich**
fixes #5235

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def _line_plot_onselect(xmin, xmax, ch_types, info, data, times, text=None,
         plot_topomap(this_data, pos, cmap=cmap, vmin=vmin, vmax=vmax,
                      axes=axarr[0][idx], show=False)
 
-    unit = 'Hz' if psd else 'ms'
+    unit = 'Hz' if psd else 's'
```
Comment (agramfort) [inline mne/viz/evoked.py]: I suspect this is wrong if you have set time_unit param to 'ms'

**Comment by massich** [comment]
@agramfort completely agree.

```py
import numpy as np
import mne

my_evoked = mne.EvokedArray(data=np.random.rand(3, 2),
                            info=mne.create_info(ch_names=3, sfreq=.1))
my_evoked.plot(time_unit='s')
my_evoked.plot(time_unit='ms')
```
produces
image

Maybe is a silly question, but instead of setting up the label through a parameter could it be computed based on info?

**Comment by agramfort** [comment]
>
> Maybe is a silly question, but instead of setting up the label through a
> parameter could it be computed based on info?
>
no. it's a user choice to use ms or s for viz.

do you understand how to fix?

**Comment by massich** [comment]
It is fixed, isn't it? check out the plots in https://github.com/mne-tools/mne-python/pull/5242#issuecomment-393096432 Otherwhise what I did not understand was the problem.

**Comment by massich** [comment]
Actually, master does the same so I did not trigger the problem properly

**Comment by massich** [comment]
Which is fixed by the PR
```py
import os.path as op
import numpy as np
import matplotlib.pyplot as plt
import mne
data_path = mne.datasets.sample.data_path()
fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis-ave.fif')
evoked = mne.read_evokeds(fname, baseline=(None, 0), proj=True)
evoked[0].plot()
```
image

```py
evoked[0].plot(time_unit='ms')
```


image

**Comment by agramfort** [comment]
travis complains

File: mne/viz/raw.py [hunk 2]
```diff
@@ -836,7 +836,7 @@ def plot_raw_psd(raw, tmin=0., tmax=np.inf, fmin=0, fmax=np.inf, proj=False,
                     types=types, zorder='std', xlim=(freqs[0], freqs[-1]),
                     ylim=None, times=freqs, bad_ch_idx=[], titles=titles,
                     ch_types_used=ch_types_used, selectable=True, psd=True,
-                    line_alpha=line_alpha, nave=None)
+                    line_alpha=line_alpha, nave=None, time_unit='ms')
```
Comment (massich) [inline mne/viz/raw.py]: This mimics the previous behavior, but I'm not sure that this is what we want.

**Comment by massich** [comment]
If that's the case I find a better solution to either add a default to `_plot_line`

**Comment by jona-sassenhagen** [comment]
Yeah looks good from here.

**Comment by massich** [comment]
@agramfort push the green button, if happy

**Comment by agramfort** [comment]
@massich can you backport ?

**Comment by agramfort** [comment]
backported

**Pull request merged**